app/src/components/Trending/TrendingItem.tsx

```diff
@@ -0,0 +1,35 @@
+import { Card, Col, Text, Grid } from '@nextui-org/react';
+import { Link } from '@remix-run/react';
+
+import { TMDB, IFilm } from '~/models/tmdb.types';
+
+interface ITrendingItemProps {
+  item: IFilm;
+}
+
+const TrendingItem = ({ item }: ITrendingItemProps) => {
+  const title = item.title || item.original_name || 'Not found';
+  const posterUrl = TMDB.poster(item.poster_path, 'w500');
+
+  return (
+    <Grid md={3} sm={3} xs={2}>
+      <Link to="/movie">
```

Comment:
dòng ni m cho Link to={props_name}

Reply:
C'est quoi ce props_name :v ?

Reply:
@ctuanle tên ngẫu nhiên do m đặt thui :v hoặc m cứ để rứa cũng đc, sau cần xài lại thì sửa sau cũng đc

Reply:
à, t cứ tưởng nó có mặc định như props với children đồ rứa :v, ok ý là dùng biến thay string á hầy

Reply:
@ctuanle ừ kiểu rứa, tại component ni xài như kiểu component ui rứa, sau ni có thể chia trường hợp render ra lm thêm cái banner nhỏ nhỏ xài trong list-film bằng component ni luôn

![image](https://user-images.githubusercontent.com/54718468/173205204-3bca921d-7af1-4696-84d8-15afa177acc4.png)